packages/cloud_firestore/lib/src/document_reference.dart

```diff
@@ -34,16 +34,18 @@ class DocumentReference {
   String get documentID => _pathComponents.last;
 
   /// Writes to the document referred to by this [DocumentReference]. If the
-  /// document does not yet exist, it will be created. If you pass [SetOptions],
-  /// the provided data will be merged into an existing document.
-  Future<void> setData(Map<String, dynamic> data, [SetOptions options]) {
+  /// document does not yet exist, it will be created.
+  ///
+  /// If [merge] is true, the provided data will be merged into an
+  /// existing document.
```

Comment:
Please document the default semantics: overwrite?

Reply:
Done